protocols/mdns/src/lib.rs

```diff
@@ -42,10 +42,10 @@ mod behaviour;
 pub use crate::behaviour::{GenMdns, MdnsEvent};
 
 #[cfg(feature = "async-io")]
-pub use crate::behaviour::Mdns;
+pub use crate::behaviour::async_io;
 
 #[cfg(feature = "tokio")]
-pub use crate::behaviour::TokioMdns;
+pub use crate::behaviour::tokio;
```

Comment:
If we already do this, should we just go all the way and rename `Mdns` to `Behaviour`, following #2217?

Reply:
yeah, agree! :)

Reply:
done :)